resources/src/main/java/org/robolectric/res/android/ResXMLParser.java

```diff
@@ -35,584 +35,577 @@
 
 public class ResXMLParser {
 
-  static final int SIZEOF_RESXMLTREE_NAMESPACE_EXT = 4;
```

Comment:
@hoisie Could those files can be formatted by google-java-format? If yes, what about formatting it in a single PR and could you help to format those files? I have noticed you have formatted some Java files separately.

Reply:
sorry If I'm mistaken but this [ResXMLParser.java](https://github.com/robolectric/robolectric/pull/7353/files/afd6132ee1dbe36321930399e2692f24fae85768#diff-1d1a02e0e0e0ef4578fccfe8247dcd1a3ef0c42d5403efcfde93f15da323ac30) is no longer needed right as we are going to keep the Kotlin version right?

Reply:
Not. We don't want to convert any code released to user/developer to Kotlin now, and only for tests. There are two implementation for resource supporting, legacy one and re-implementation libandroidfw resource logic in Java. This class is used to parse XML of resources in test apk. If you grep `ResXMLParser`, you can find many references to this class.

Reply:
ahh okay, but [ResXMLParser.java](https://github.com/robolectric/robolectric/pull/7353/files/afd6132ee1dbe36321930399e2692f24fae85768#diff-1d1a02e0e0e0ef4578fccfe8247dcd1a3ef0c42d5403efcfde93f15da323ac30) this is not deleted its just formatted according to the google format thingy

Reply:
I'm contacting with @hoisie to discuss how to process errorprone fixing brings many formatting changes. It's increase the difficulty to review PR. The formatting problem increase the size of a small PR a lot.

Reply:
ohh, yea you are right

Reply:
TBH, the AnnotateFormatMethod warning is lower priority than other code related warnings, and current PR brings errorprone direct dependency(including errorprone code to this module) to resource module, and about six thousand change lines. I'm not very comfortable with current PR.

Reply:
Yeah, right. for now I can just push the code with the warning-related fix and leave the formatting changes. and we can keep this PR on hold until I will fix the other significant priority warnings.

Reply:
@utzcoz How about if we make it this way, I will make 2 commits, one for warnings and one for refactoring.
During the review, you have to only carefully look at commit 1 and commit 2 you obviously take a look at it too, it's a kind of separation.
Then later before merging, I will squash those two commits into one
In this approach, we have to trust commit 2, it only contains refactoring related things.
but still, it's better than what we are doing now I think.